diff --git a/src/Search/QueryBuilder.php b/src/Search/QueryBuilder.php
index c715cfc..b05598f 100644
--- a/src/Search/QueryBuilder.php
+++ b/src/Search/QueryBuilder.php
@@ -21,7 +21,7 @@ use Doctrine\ORM\QueryBuilder as DoctrineQueryBuilder;
 class QueryBuilder
 {
     /** @var Registry */
-    private $doctrine;
+    protected $doctrine;
 
     public function __construct(Registry $doctrine)
     {
